src/sklearn_evaluation/plot/regression.py

```diff
@@ -118,9 +119,14 @@ def prediction_error(y_true, y_pred, model=None, ax=None):
         raise TypeError(
             '"fit_intercept" attribute not in model. ' "Cannot plot prediction error."
         )
+
+    if isinstance(y_true, pd.Series):
+        y_reshaped = y_true.values.reshape((-1, 1))
+    else:
+        y_reshaped = y_true.reshape((-1, 1))
     # best fit line
-    model.fit(y_true.reshape((-1, 1)), y_pred)
-    x = np.linspace(80, 230, 100)
+    model.fit(y_reshaped, y_pred)
+    x = np.linspace(min(y_true), max(y_true))
```

Comment:
let's test with a few values to see if this yields good results

Reply:
I haven't made this change as of now. Fixed the other comments. 
I have raised this [issue](https://github.com/ploomber/sklearn-evaluation/issues/159). 
I think this plot needs some rework. Should we take it up after merging the regression quickstart? 
Thoughts ? @edublancas

Reply:
I'll leave it up to you. If you think we need to fix the plot before merging this one, that's fine. Otherwise, we can merge this and then fix the plot. What do you think it's best? let me know so I know If I should do another pass to this PR or wait until the plot is fixed.

Reply:
I think it's better to fix before the merge.

Reply:
Please submit a PR?